src/vivarium/framework/engine.py

```diff
@@ -86,8 +86,6 @@ def __init__(self, model_specification: Union[str, Path, ConfigTree] = None,
         for name, controller in self._plugin_manager.get_optional_controllers().items():
             setattr(self, f'_{name}', controller)
 
-        self._resource.add_group('initialization')
```

Comment:
I fused the resource group and resource manager since the only resource group we were creating was this one.  So this goes away.